manifests/cf-manifest/test-config/acceptance_tests.yml

```diff
@@ -8,22 +8,25 @@ test_password: ((secrets_test_user_password))
 skip_ssl_validation: false
 backend: "diego"
 skip_diego_unsupported_tests: true
-include_tasks: true
-include_v3: true
-include_security_groups: true
-include_routing: true
+include_docker: true
 include_internet_dependent: true
 include_logging: true
 include_operator: true
+include_route_services: true
+include_routing: true
+include_security_groups: true
 include_service_discovery: true
 include_services: true
 include_ssh: true
-include_docker: true
-include_route_services: true
+include_tasks: true
+include_v3: true
 include_zipkin: true
 artifacts_directory: "/tmp/artifacts"
 use_existing_user: false
 include_container_networking: true
 name_prefix: ((name_prefix))
+include_isolation_segments: false
+isolation_segment_name: dev-1
```

Comment:
Unsure if this really wants to be hardcoded to `dev-1`

Reply:
resolved in ffa078f